test/runtests.jl

```diff
@@ -66,10 +66,15 @@ close(gzfile)
 # Screw up the file
 raw_file = open(test_compressed, "r+")
 seek(raw_file, 3) # leave the gzip magic 2-byte header
-write(raw_file, zeros(Uint8, 10))
+write(raw_file, zeros(UInt8, 10))
 close(raw_file)
 
-@test_throws ArgumentError gzopen(readall, test_compressed)
+try
+    gzopen(readall, test_compressed)
+    throw(Error("Expecting an ArgumentError or similar"))
+catch e
+    @test isa(e, ArgumentError) || contains(e.msg, "too many arguments")
+end
 
```

Comment:
I've got that -- just confused why I'm getting those locally and `ZError`s on Travis for both 0.3. and 0.4. Could add more logic in the catch block, just hoping I could avoid haphazardly papering over an issue.